fili-core/src/test/groovy/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProviderSpec.groovy

```diff
@@ -98,6 +137,87 @@ class LuceneSearchProviderSpec extends SearchProviderSpec<LuceneSearchProvider>
         1 * keyValueStore.put('cardinality_key', '14')
     }
 
+    def "moveDirEntries moves all entries of a directory to a new directory, while keeping all old empty dirs"() {
+        expect:
```

Comment:
Is it necessary to have both `expect` and `then` steps at the same time?

Reply:
If you take a look at logic of this test, you will see that this test first makes sure `sourcePath` contains all files to be removed and `destinationPath` doesn't, which is what `expect` is all about, then the test makes sure, after `moveDirEntries`, `sourcePath` doesn't contain the files and `destinationPath` contains all the files, because all files have been moved from `sourcePath` to `destinationPath`, which is that `then` is all about.

Reply:
If you take a look at logic of this test, you will see that this test first makes sure `sourcePath` contains all files to be removed and `destinationPath` doesn't, which is what `expect` is all about, then the test makes sure, after `moveDirEntries`, `sourcePath` doesn't contain the files and `destinationPath` contains all the files, because all files have been moved from `sourcePath` to `destinationPath`, which is that `then` is all about.